WordPress/wp20.wordpress.net
# Update theme layouts and styles

See #2 

This PR:

1. Updates content and images
2. Updates typography, colors and spacing
3. Removes unnecessary styles (IE8, supports, etc)

NOTE: content for What's On and Live is not functional yet and requires further work in #4 and #3, main nav and locale switcher requires #1, and Swag links are empty, pending #5. There are more refinements to be made to pagination and mobile views.

## Screenshots

### Desktop

| What's on | News | Single | Live | Swag |
|-|-|-|-|-|
| ![localhost_8888_(Desktop)](https://user-images.githubusercontent.com/1017872/225180216-6f9c76be-2331-4c0a-b5d2-4e9792d409d6.png) | ![localhost_8888_news_(Desktop) (1)](https://user-images.githubusercontent.com/1017872/225180245-969d3cab-a1da-4311-9e09-3f642feb232a.png) | ![localhost_8888_wp20-commemorative-jones-soda-bottles-design-yours_(Desktop) (1)](https://user-images.githubusercontent.com/1017872/225180281-1931639e-9e8e-494c-a486-d181fbe2bbef.png) | ![localhost_8888_live_(Desktop) (1)](https://user-images.githubusercontent.com/1017872/225180373-32ccd66d-c766-4c39-a8ba-88a6a033aa30.png) | ![localhost_8888_swag_(Desktop) (1)](https://user-images.githubusercontent.com/1017872/225180385-792c5a9b-4bed-4260-a6dc-58bdc2284c53.png) |

### Mobile

| What's on | News | Single | Live | Swag |
|-|-|-|-|-|
| ![localhost_8888_(Samsung Galaxy S20 Ultra) (2)](https://user-images.githubusercontent.com/1017872/225180623-6b1f9a71-4405-4508-8f99-227f7c486007.png) | ![localhost_8888_news_(Samsung Galaxy S20 Ultra)](https://user-images.githubusercontent.com/1017872/225180668-5adb0d4e-a096-4ab8-9ade-eaf9fdc8c0e2.png) | ![localhost_8888_wp20-commemorative-jones-soda-bottles-design-yours_(Samsung Galaxy S20 Ultra)](https://user-images.githubusercontent.com/1017872/225180647-83d208ea-61bc-42a5-a944-869153c7c28e.png) | ![localhost_8888_live_(Samsung Galaxy S20 Ultra)](https://user-images.githubusercontent.com/1017872/225180698-f4dcd540-5838-45bb-bbb0-106be28b9b53.png) | ![localhost_8888_swag_(Samsung Galaxy S20 Ultra)](https://user-images.githubusercontent.com/1017872/225180712-213cb96c-b5cb-4a4e-a2c2-cc78500a4918.png) |

## How to test

1. Follow the [setup instructions](https://github.com/WordPress/wp20.wordpress.net/tree/trunk#readme)
2. Navigate the pages (including single post); check layout against designs above, responsive behaviour, a11y.
